supervisor/supervisor.py

```diff
@@ -50,7 +51,10 @@ def ip_address(self) -> IPv4Address:
     @property
     def need_update(self) -> bool:
         """Return True if an update is available."""
-        return self.version != self.latest_version
+        try:
```

Comment:
```suggestion
		if self.version == "DEV":
			return False
        try:
```

Let's add a check for DEV as the version here to stop it from offering updates when you start from git

Reply:
I like that. Let me write it self on vscode

Reply:
If you do that, maybe add it as a const